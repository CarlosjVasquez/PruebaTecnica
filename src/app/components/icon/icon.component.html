<picture>
  <img [src]="src" alt="icon" [width]="size" [height]="size" />
</picture>
