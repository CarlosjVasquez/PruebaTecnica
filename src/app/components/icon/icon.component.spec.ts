import { ComponentFixture, TestBed } from '@angular/core/testing';

import { IconComponent } from './icon.component';

describe('IconComponent', () => {
  let component: IconComponent;
  let fixture: ComponentFixture<IconComponent>;
  let compiled: HTMLElement;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      providers: [IconComponent],
    }).compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(IconComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
    compiled = fixture.nativeElement;
  });

  test('should create', () => {
    expect(component).toBeTruthy();
  });

  test('match to snapshot', () => {
    expect(compiled).toMatchSnapshot();
  });
});
