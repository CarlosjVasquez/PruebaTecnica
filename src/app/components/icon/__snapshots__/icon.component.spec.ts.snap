// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconComponent match to snapshot 1`] = `
<div
  id="root1"
>
  <picture>
    <img
      alt="icon"
      height="25"
      src=""
      width="25"
    />
  </picture>
</div>
`;
